debian/rules

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+export DH_VERBOSE := 1
+export PATH := /usr/lib/go-1.11/bin/:$(PATH)
```

Comment:
hardcoded go version? Won't there be a problem?

Reply:
yeah, we have also hardcoded dependency to this version:
https://github.com/mysteriumnetwork/node/pull/1073/files#diff-d837a1c17de0268bcea321239ddbed47R8